src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -2121,33 +2133,107 @@ private GencodeFuncotation createIgrFuncotation(final VariantContext variant,
     }
 
     /**
-     * Creates a {@link GencodeFuncotation}s based on a given spanning deletion {@link Allele}.
+     * Creates a {@link GencodeFuncotation} based on a given symbolic alternate {@link Allele}.
```

Comment:
Can we have one more sentence here? Such as "Creates a COULD NOT DETERMINE variant classification."

Reply:
Sure.  Fixed!